plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -201,6 +201,16 @@ class CreateServiceModalForm extends Component {
       return true;
     }
 
+    if (nextProps.expandAdvancedSettings) {
+      nextProps.resetExpandAdvancedSettings();
```

Comment:
to me kicking off a side effect in `shouldComponentUpdate` (which has the only purpose of figuring out whether the component should be rerendered or not) does not feel good.

Reply:
Where do you prefer to have it in this case?

Reply:
I moved it to `componentDidUpdate`.